cvmfs/server/cvmfs_server_util.sh

```diff
@@ -914,7 +914,7 @@ minpidof() {
 
 
 is_systemd() {
```

Comment:
Is this function still named correctly?  If I understand correctly this will now  always prefer supervisord if the binary is available. Could you maybe add a comment to clarify this?

Reply:
Yes it is still named correctly.  It returns true if systemd is to be used.  It's really not a change in the way it worked before; previously it returned true if the service command was not available, and now it returns true if neither the service command or supervisorctl command are available, defaulting to systemd.  I will add a comment although it seems pretty obvious to me.

Reply:
Oh I guess it is subtly different, in that the service command is only searched for if systemctl is missing.